sockeye/transformer.py

```diff
@@ -399,9 +401,10 @@ def backward(self, req, out_grad, in_data, out_data, in_grad, aux):
 @mx.operator.register("auto_regressive_bias")
 class AutoRegressiveBiasProp(mx.operator.CustomOpProp):
 
-    def __init__(self, length: str) -> None:
+    def __init__(self, length: str, dtype: Optional[str] = C.DTYPE_FP32) -> None:
```

Comment:
I don't think `dtype` could be None, so `Optional` does not apply here.

Reply:
fixed